meter.go

```diff
@@ -15,6 +15,7 @@ type Meter interface {
 	Rate15() float64
 	RateMean() float64
 	Snapshot() Meter
+	Stop()
```

Comment:
I would add the same method on `Timer` as they rely on a `Meter` under the hood.

I am even wondering if having an interface `Stoppable` implemented by `Timer` and `Meter` and invoking `Stop` in `Registry.Unregister(string)` when the metric implements such an interface is the proper way to clean those metrics that can leak (potentially supporting similar behavior for custom metrics).

What do you think?

Reply:
Rephrasing just so I understand: you're suggesting we add the interface and implementations, then leave it up to the user to call `Unregister` when a metric has outlived its usefulness, correct? I suppose there's not really a magical way we can know when it's safe to do so internally.

Reply:
@mihasya Absolutely.

`Unregister` (and `UnregisterAll` by extension) would check if the underlying `interface{}` metric implements `Stoppable` and call `Stop()` as like you said there is no way to know when those metrics are not used anymore.